src/tools/add-timestamp-for-log.ts

```diff
@@ -0,0 +1,18 @@
+export function addTimestampForLog(log: string, arg?: any) {
+  let timestamp = changeTimeFormat();
+  if(arg) {
```

Comment:
I don't know why you do this, please do research to see how other guys do.
The purpose is we should keep all inputs just as console.log.